.github/workflows/ci.yml

```diff
@@ -111,26 +111,14 @@ jobs:
     secrets:
       pypi_token: ${{ secrets.pypi_token }}
 
-  notify_pass:
+  notify:
     if: github.event_name == 'workflow_dispatch'
```

Comment:
Needs `always()` otherwise it would only message on success:
```suggestion
    if: always() && github.event_name == 'workflow_dispatch'
```

Reply:
So actions will not run this without this?

Reply:
We want the `notify` job to run even if its dependants (`[publish, online, cron]`) fail. See https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-not-requiring-successful-dependent-jobs

Reply:
Ah yes, I fixed this in ndcube but forgot to put it here as well.